baseplate/sidecars/live_data_watcher.py

```diff
@@ -25,6 +25,8 @@
 
 class NodeWatcher:
     def __init__(self, dest: str, owner: int, group: int, mode: int):
+        pathlib.Path(dest).parent.mkdir(parents=True, exist_ok=True)
+
```

Comment:
It's kind of odd to me to have this in the `__init__`, but it seemed to work best here. It sort of makes sense that the object which writes to a file should be concerned that the destination directory exists. I also wanted to avoid doing this each time there is a change, as this needs to be done at most once per execution.

Reply:
I think we might actually want to do this every time. What happens if the directory is deleted?

Reply:
That is true, I hadn't thought of that :+1: I'll move it.